regexes/client/mobile_apps.yml

```
@@ -57,6 +57,11 @@
   name: 'WeChat'
   version: '$1'
 
+# DingTalk
+- regex: 'DingTalk/([^\)]+)'
```

Might be better to match the version number with digits instead of all chars except of `)`. Otherwise it might match too much if the identifier changes and is no longer surrounded by brackets...

```suggestion
- regex: 'DingTalk/([0-9\.]+)'
```